samples/drivers/adc/boards/mikroe_adc_click.overlay

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2023 Vestas Wind Systems A/S
+ */
+
+/ {
```

Comment:
I believe that if adding this overlay then we should have en extra entry in sample.yml to ensure combo is picked up by twister: https://github.com/zephyrproject-rtos/zephyr/blob/b98d2a1b4744939a8a3259f112da90b3e4c648f7/samples/drivers/adc/sample.yaml#L1